tz.cpp

```diff
@@ -218,7 +218,12 @@ static std::string get_install()
 
 #ifndef INSTALL
 
-static const std::string install = get_install();
+static std::string install = get_install();
+
+void set_install(std::string install_folder)
+{
+   install = install_folder;
+}
```

Comment:
This is an open question that I do not yet have a good answer for (sorry).

The ability to set the `install` at run time opens up the possibility of changing the `install` after, or even during the database initialization.  This is not an insurmountable problem.  But it is a problem I have not yet addressed to my own satisfaction.